spec/commands/updater_spec.rb

```diff
@@ -10,19 +10,21 @@
   end
 
   describe "#run!" do
-    before do
-      FileUtils.rm_rf("/tmp/plinytest")
-      FileUtils.mkdir_p("/tmp/plinytest")
-      Dir.chdir("/tmp/plinytest")
-      File.open("/tmp/plinytest/Gemfile.lock", "w") do |f|
-        f.puts "    pliny (0.6.3)"
+    around do |example|
+      Dir.mktmpdir("plinytest-") do |dir|
+        Dir.chdir(dir) do
+          File.open("./Gemfile.lock", "w") do |f|
```

Comment:
Not a bit deal really, but just wondering if you actually need the `./` bit here..? (Same comment applies to all the other instances in this PR.)

Reply:
Probably not...? I did it in hopes of being explicit that we're talking about paths from right where we are. Which is the same thing that not having the dot should do, but I feel like sometimes that nuance gets lost when glancing at the code. Happy to drop the `./` if you prefer.

Reply:
Meh nah, I'm not that bothered about it and it sounds like a good reason. All good 👍